Project 8 - Mapty/css/style.css

```diff
@@ -0,0 +1,320 @@
+/* CSS for Mapty */
+:root {
+  --color-brand--1: #45ecff;
+  --color-brand--2: #00c46a;
+
+  --color-dark--1: #2d3439;
+  --color-dark--2: #42484d;
+  --color-light--1: #aaa;
+  --color-light--2: #ececec;
+  --color-light--3: rgb(214, 222, 224);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+html {
+  font-size: 62.5%;
+  box-sizing: border-box;
```

Comment:
@sknepali We can remove this if we add border-box in universal selector

Reply:
Yes, we can do it